source/isaaclab_assets/isaaclab_assets/robots/agility.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2022-2025, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import pathlib
+from typing import Literal
+
+import isaaclab.sim as sim_utils
+from isaaclab.actuators import IdealPDActuatorCfg, ImplicitActuatorCfg
+from isaaclab.assets.articulation import ArticulationCfg
+from isaaclab.utils.assets import ISAAC_NUCLEUS_DIR
+
+FILE_DIR = pathlib.Path(__file__).parent
+
+
+def get_digit_articulation_cfg(
```

Comment:
I would suggest to not have this functional way of asset configuration. Instead it is better to just have explicit configurations for each style (for consistency with the rest of the examples).